src/deps.txt

```diff
@@ -85,8 +85,8 @@ python3-jsonschema
 # automatically
 coreos-installer
 
-# For the ability to easily pass in an fcc to kola
-fcct
+# For the ability to easily pass in a butane config to kola
```

Comment:
nit: We generally capitalize Butane except when talking about the command itself.